src/target/stm32f1.c

```diff
@@ -142,8 +149,10 @@ bool stm32f1_probe(target *t)
 	case 0x439:  /* STM32F302C8 */
 	case 0x446:  /* STM32F303xD/E and STM32F398xE */
 		t->driver = "STM32F3";
+		target_add_ram(t, 0x10000000, 0x4000);
 		target_add_ram(t, 0x20000000, 0x10000);
-		stm32f1_add_flash(t, 0x8000000, 0x80000, 0x800);
+		flash_size= (target_mem_read32(t, FLASHSIZE) & 0xffff) * 0x400;
```

Comment:
Why this change?
- It's not declared in the PR description or the commit message.
- It appears to be inconsistent with your newly added comment above, that also has nothing to do with CCM RAM.
- Put spaces on both sides of `=`.
- `FLASHSIZE` is 0x1FFFF7E0, but the register is at 0x1FFFF7CC, is it not?  Did you test this?